astro/src/content/docs/lifecycle/authenticate-users/identity-providers/enterprise/samlv2-idp-initiated.mdx

```diff
@@ -70,35 +70,50 @@ This will take you to the <strong>Add SAML v2 IdP Initiated</strong> panel. Here
+    To configure a lambda, navigate to <strong>Settings -> Lambdas</strong>.
   </APIField>
-  <APIField name="Verification key" required>
-    The public key or certificate that you must import into FusionAuth's KeyMaster. This is the public key provided to you by the identity provider.
+  <APIField name="Debug enabled" optional>
+    Some identity providers are not compliant with the SAML and XML signing specifications. This makes it challenging to get them working with FusionAuth.
+
+    If you are running into integration issues, toggle this setting on and FusionAuth will output debugging information into the Event Log during a SAML login. You can find the event log in <strong>System -> Event Log</strong>.
   </APIField>
 </APIBlock>
 
 ### Options
 
-<img src="/img/docs/lifecycle/authenticate-users/identity-providers/enterprise/identity-provider-samlv2-idp-initiated-add-options.png" alt="Add SAML v2 Options section" width="1200" role="bottom-cropped" />
+![Add SAML v2 Options section](/img/docs/lifecycle/authenticate-users/identity-providers/enterprise/identity-provider-samlv2-idp-initiated-add-options.png)
 
 #### Form Fields
 
 <APIBlock>
-  <APIField name="Reconcile lambda" optional>
-    A lambda maps custom claims returned from the SAML response into the FusionAuth `User` and `Registration`. To learn more about creating a lambda, view the [SAML v2 Reconcile lambda documentation](/docs/extend/code/lambdas/samlv2-response-reconcile).
-
-    To configure a lambda, navigate to <strong>Settings -> Lambdas</strong>.
+  <APIField name="Unique Id claim" optional>
+    The name of the claim that is returned in the SAML response that contains the unique Id of the user.
   </APIField>
-  <APIField name="Debug" optional>
-    Some identity providers are not compliant with the SAML and XML signing specifications. This makes it challenging to get them working with FusionAuth.
+  <APIField name="Use NameId for email" optional>
+    If this is enabled, FusionAuth will assume that the `NameID` in the SAML response contains the email address of the user.
+  </APIField>
+  <APIField name="Email claim" required>
+    The name of the email claim returned in the SAML response.
 
-    If you are running into integration issues, toggle this setting on and FusionAuth will output debugging information into the Event Log during a SAML login. You can find the event log in <strong>System -> Event Log</strong>.
+    When <InlineField>Use NameId for email</InlineField> is enabled this field will not be displayed and will not be required.
+  </APIField>
+  <APIField name="Username claim" optional>
+    The name of the username claim returned in the SAML response.
+  </APIField>
+  <APIField name="Require encrypted assertions" optional since="1.55.1">
+    When enabled FusionAuth requires encrypted assertions in SAML responses from the identity provider. SAML responses from the identity provider containing unencrypted assertions will be rejected by FusionAuth.
```

Comment:
I would personally put a comma after `When enabled` but it's so inconsistent across the documentation that this one doesn't really matter.